Lib/xmlrpc/server.py

```diff
@@ -894,7 +895,7 @@ def generate_html_documentation(self):
                                 methods
                             )
 
-        return documenter.page(self.server_title, documentation)
+        return documenter.page(html.escape(self.server_title), documentation)
```

Comment:
Please add a test for this in `Lib/test/test_docxmlrpc.py` . Test could be to set a script/html content using `self.serv.set_server_title` and ensure the documentation returned as html response from the server has escaped content inside the title tag.

Reply:
@tikarthi Oh thanks I will add it :)